content/500-platform/550-platform-cli/about.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'About'
+metaTitle: 'Platform CLI: About'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This documentation provides details on how to access the Prisma Data Platform using the Prisma CLI. Follow the [installation guide](/orm/tools/prisma-cli#installation) for the Prisma CLI to start using it.
+
+The integration, currently in [Early Access](/platform/maturity-levels#early-access), extends the functionality of the Prisma CLI, enabling you to manage your Platform projects. For the complete list of available commands, navigate to the [Commands page](/platform/platfrom-cli/commands).
```

Comment:
the commands link returns a 404, https://docs-git-platform-cli-prisma.vercel.app/platform/platfrom-cli/commands

Reply:
There was a typo in the original filename; the link wasn't updated after changing the filename; great catch 💯